tests/test_edgeql_insert.py

```diff
@@ -6473,6 +6487,7 @@ async def test_edgeql_insert_conditional_01(self):
             variables=([True, False],)
         )
 
+        # XXX: THIS IS FAILING BUT THAT IS BULLSHIT
```

Comment:
Why would this be failing?

Which prefix is being factored out?

Reply:
I think it's something hinky with how IF is compiled for DML.

Hm.

Reply:
Ah, but more important, it is *not* failing. I fixed it (because it seemed critically bad), and never removed the comment.